docker/elasticsearch.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.8'
+
+services:
+  uwsgi:
```

Comment:
it works as a "override" so no need to replicate what already is in the `default.yml`

just uwsgi, depend_on and elasticsearch are enough

Reply:
Do I need to mention ```postgres``` in ```depends_on``` again too?